### ISSUE TITLE
How exactly does the plugin work on iOS?

### ISSUE THREAD

**Issue by frederikheld** [capacitor-community/safe-area#39]
This is not exactly a bug, but a question to @tafelnl about unclear docs.

In [this comment](https://github.com/capacitor-community/safe-area/pull/2#issuecomment-2092723752) you say

> on web and iOS the safe area insets are working correctly out of the box. So there's no need patching it for those platforms.

The [README](https://github.com/capacitor-community/safe-area/blob/master/README.md) is a bit more specific about this

> On web and iOS the safe area insets work out of the box1. That is, on these platforms the env(safe-area-inset-*) CSS variables will have the correct values by default.

And it also says:

> There's one small but important quirck though, since we cannot override the native `env(safe-area-inset-*)` variables, the values are instead written to custom `var(--safe-area-inset-*)` variables

So my question is: will the safe area insets be available on iOS as `var(--safe-area-inset-*)` as well, or do I need to use `env(safe-area-inset-*)` there?

In other words: can I use the same CSS for both platforms?

**Comment by felixgeissler** [comment]
As I bumped into this topic as well, here is what I did wrong:
I assumed that adding the `import '@capacitor-community/safe-area';` to my _capacitor.config.ts_ would be the proper way of initializing this plugin, according to the [Enable by using the Configuration](https://github.com/capacitor-community/safe-area#enable-by-using-the-configuration) section in the readme. But while Android worked out fine, I had trouble with iOS. After a bit of testing I noticed that on iOS it will not initialize automatically when imported via the capacitor config.

I resolved this by moving the import to my `main.ts`. After this change both iOS and Android initialized automatically.

**Comment by felixgeissler** [comment]
Yes, that makes sense! Maybe this should be mentioned in the readme though.

I just discovered while testing the import in `main.ts` that I can now reproduce the race condition described in #40
